ui/cypress/e2e/spec.cy.ts

```diff
@@ -62,15 +62,15 @@ describe('devfile editor spec', () => {
     cy.getByDataCy('container-env-value-2').type("val3");
 
     cy.getByDataCy('volume-mount-add').click();
-    cy.getByDataCy('volume-mount-path-0').type("/mnt/vol1");
+    cy.getByDataCy('volume-mount-path-0').type("/mnt/vol1", {force: true});
```

Comment:
Not sure why, but I had to make Cypress forcibly bypass some checks with `force: true`. Otherwise, Cypress would complain that the element was covered by another one:

```
CypressError: Timed out retrying after 4000ms: `cy.type()` failed because this element:

`<input _ngcontent-xcd-c181="" formcontrolname="path" matinput="" class="mat-mdc-input-element ng-tns-c84-53 ng-untouched ng-pristine ng-invalid mat-mdc-form-field-input-control mdc-text-field__input cdk-text-field-autofill-monitored" ng-reflect-name="path" data-cy="volume-mount-path-0" id="mat-input-32" required="" aria-required="true">`

is being covered by another element:

`<span aria-hidden="true" class="mat-mdc-form-field-required-marker mdc-floating-label--required ng-tns-c84-53 ng-star-inserted"></span>`

Fix this problem, or use {force: true} to disable error checking.
```